pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -123,18 +130,63 @@ func verifyDevicePath(diskUUID string) string {
 }
 
 func probeAttachedVolume() error {
```

Comment:
Please add some doc comment here explaining what's going on for people not familiar w/ vsphere -- are brief description is OK.  Why does this method need to write out a file?